client/src/actions/index.js

```diff
@@ -19,6 +19,7 @@ function receiveURL(html) {
     };
 }
 
+// why do you return function here?
```

Comment:
Hi,
Thanks for your review.
This is my bad, maybe the first time I want to return a function here, then I change some thing and forgot updating for this function